### PR TITLE
226 remove bvv

### LINKED ISSUE #226: Removing the BVV IP field in the GUI
I integrated the BVV node into the cashier node, and also added support to dynamically fetch and assign bvv when the dealer node starts. The portion to be removed is highlighted in the below diagram.
image

### PR DESCRIPTION
**Pull request by piggydoughnut**
Resolves #226 

I removed BVV from the game modal and the WS setup for listening to messages from BVV. 

Frontend is completely disconnected from BVV now.

### CODE REVIEW

File: src/store/testState.ts [hunk 1]
```diff
@@ -143,7 +140,8 @@ const initialState: IState = {
   // Where does the user sit
   userSeat: "player1",
   // The player that won the game
-  winners: [undefined]
+  winners: [undefined],
+  withdrawAddressList: []
```
Comment (norbertdragan) [inline src/store/testState.ts:144]: What about this line? Is this related?

Comment (piggydoughnut) [reply src/store/testState.ts:144]: ah, the linter or smth was complaining that this is missing in that file :)
